crates/sui-config/data/fullnode-template-with-path.yaml

```diff
@@ -18,6 +18,7 @@ authority-store-pruning-config:
   max-checkpoints-in-batch: 10
   max-transactions-in-batch: 1000
   pruning-run-delay-seconds: 60
+  enable-pruning-tombstones: false
```

Comment:
do we need to introduce a new setting?
our pruning config already looks overbloated
Or is the plan to run few experiments and eventually deprecate it?

Reply:
This is currently needed for 2 reasons:

1, for experiment control
2, we can't prune wrapped tombstone without `simplified_unwrap_then_delete` is enabled.

If both are resolved, we can remove this setting later.